gradle/libs.versions.toml

```
@@ -1,38 +1,36 @@
 [versions]
-androidGradlePlugin = "8.1.1"
-androidxActivity = "1.5.1"
-androidXBrowser = "1.5.0"
-androidxCompose = "1.3.0"
-androidxComposeCompiler = "1.4.6"
-androidxCore = "1.9.0"
-androidxCoreTesting = "2.1.0"
-androidxEspresso = "3.4.0"
-androidxLifecycle = "2.3.1"
-androidxLifecycleRuntimeCompose = "2.6.0-beta01"
-androidxTestExt = "1.1.2"
-androidxViewmodelCompose = "2.6.1"
-compileSdk = "33"
-compose-material3 = "1.1.0"
+androidGradlePlugin = "8.2.0"
+androidxActivity = "1.8.1"
+androidXBrowser = "1.7.0"
+composeBOM = "2023.10.01"
+androidxComposeCompiler = "1.5.4"
+androidxCore = "1.12.0"
+androidxEspresso = "3.5.1"
+androidxLifecycle = "2.6.2"
+androidxLifecycleRuntimeCompose = "2.7.0-rc01"
+androidxTestExt = "1.1.5"
+androidxViewmodelCompose = "2.6.2"
+compileSdk = "34"
 junit = "4.13.2"
-kotlin = "1.8.20"
+kotlin = "1.9.20"
```

Related to the comment above for compose compiler, I think this should be 1.9.21:
```suggestion
kotlin = "1.9.21"
```